### ISSUE TITLE
handleHelperFunction.js file

### ISSUE THREAD

**Issue by Karyum** [FACN3/swearUrban#32]
So there is a bit of functional programming going on here which is great but don't you think putting all the functions into 1 object is hard to read and hard to follow the code around ??

isn't it better to just make your lives easier by splitting the functions into files ? :sweat_smile: 

that way everything is readable and organized and can be altered anytime you want without any fuck ups :+1:

**Comment by hasansaad1** [comment]
or we can just do an interface for the object? maybe , because the three functions deal with the same subjects maybe we also need to change the objects name ... :smile:

**Comment by Karyum** [comment]
@hasansaad1 So why not split the functions entirely or even split them into files . my main question here, is it clear to you everything that is happening  in that file ? like **100%** clear? if not how do you think you can make it more readable ?
